lib/Host.php

```diff
@@ -57,14 +65,18 @@ public function expose(string $address, int $port): Host {
     /**
      * Assign a domain name (e.g. localhost or mysite.com or subdomain.mysite.com).
      *
-     * A host name is only required if a server exposes more than one host. If a name is not defined
-     * the server will default to "localhost"
+     * An explicit host name is only required if a server exposes more than one host. If a name is
+     * not defined (or "*") the server will allow any hostname.
```

Comment:
That's not true, no? If two different ports are used, no names are required?

Reply:
\* on a same interface. Yeah, that's missing.